### PR TITLE
small bugs

### PR DESCRIPTION
**Pull request by zuzi-m**
BatchComparison.java - returned the use of `node_average` to make it possible to show average face
NavigatorTopComponent.java - there was a bug in displaying the average face node anyways, so fixed its condition. Also, for the "result" node it always makes sure to display the average model, not the last model displayed
StartingPanel.java - removed unnecessary code that did nothing. Its original purpose (deleting the temporary files folder) is now replaced by this https://github.com/Fidentis/Analyst/blob/development/Utils/src/cz/fidentis/utils/FileUtils.java#L130-L133 Also, now the code makes sure to create a separate, unique temporary folder for each project created

### CODE REVIEW

File: GUI/src/cz/fidentis/gui/StartingPanel.java [hunk 1]
```diff
@@ -304,28 +298,7 @@ public static void createNewProject() {
...
-                
-                    Thread t = new Thread(run);
-                    t.start();                    
-                
-                    project.setTempDirectory(FileUtils.instance()
-                        .createTMPmoduleFolder(String.valueOf(System.currentTimeMillis())));
-                
-                    master.updateObservers();
-                }
+                project.setTempDirectory(FileUtils.instance().createTMPmoduleFolder(String.valueOf(System.currentTimeMillis())));
```
Comment (ZueFe) [inline GUI/src/cz/fidentis/gui/StartingPanel.java:301]: len pripomienka, prvy krat ked sa zavola FileUtils.instance() vymaze sa predchadzajuci tmp folder, ak tam ostal z predchadzajucich behov fidentisu a vytvori sa nova zlozka pre tento beh programu. Pokial chces mazanie spravit explicitne v triede Install (tusim), tak to tam nezabudni hodit predtym, ako sa mergne PR (za okolnosti, ked bude tych suborov na mazanie vela, tak teraz program zamrzne a nebude existovat ziaden sposob oznamenia uzivatelovi, ze program pracuje a nezamrzol)

**Comment by zuzi-m** [comment]
@ZueFe commitol som fix ukladanie feature points v 1:N projekte a presunul som ten kod ktory maze obsah temp priecinku. Poprosim review a ak je to ok tak merge

File: GUI/src/cz/fidentis/gui/Installer.java [hunk 2]
```diff
@@ -72,6 +72,13 @@ public void restored() {
          UIManager.put("text", new Color(0, 0, 0));}*/ catch (URISyntaxException | ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException | ConcurrentModificationException ex) {
             Exceptions.printStackTrace(ex);
         }
+        
+        // delete the temp folder if it existed before the start of the program
+        try {
+            FileUtils.instance().createTMPfolder(true);
```
Comment (ZueFe) [inline GUI/src/cz/fidentis/gui/Installer.java]: nepodarilo sa teda prist na to ako na splash screen prihodit spravu? Skus toto ci to pomoze https://forums.netbeans.org/ntopic23810.html

**Comment by ZueFe** [comment]
Vyzera to ok, ked tak skus este ten link co som posielala k splash screen msg, pokial to nepojde, tak to rovno mergni.

File: GUI/src/cz/fidentis/gui/Installer.java (hunk 2 above)
Comment (zuzi-m) [reply GUI/src/cz/fidentis/gui/Installer.java]: velmi dobry tip, islo to len bolo treba spravit o jeden krok viac ako bolo v tom postupe (includnut aj Bootstrap modul)

**Pull request merged**